src/app/shared/services/in-memory-data.service.ts

```diff
@@ -9,13 +13,12 @@ import { InMemoryDbService } from 'angular-in-memory-web-api'
 export class InMemoryDataService implements InMemoryDbService {
   createDb(): any {
     const advertisements = ADVERTISEMENTS_MOCK_DATA
-    return { advertisements }
+    const landlords = LANDLORDS_MOCK_DATA
```

Comment:
Stavo pensando che, forse nell'InMemoryDbService dovremmo avere gli array già `buildati` con il metodo `Landlord.build()`, così quando facciamo la chiamata dal servizio restituisce un observable di landlords_mock_data buildati

Reply:
In effetti può essere una buona idea grazie @Gabriele-bil 
quindi dici di cambiarlo in questo modo: `const landlords = LANDLORDS_MOCK_DATA.map((landlord) => Landlord.Build(landlord))`

A questo punto, @pc-codingavision posso fare lo stesso per gli advertisements, crearmi la classe e inserire il metodo `get score()` come abbiamo fatto nei [dati mockati creati da Fanni](https://github.com/pc-codingavision/palermo-fe-final-project/blob/develop/src/app/modules/core/advertisement/mock-advertisement/mock-advertisement.ts)?
Così da avere gli advertisements buildati con la properità score (ovviamente landlord e property rimangono id)

Reply:
@ludovico-zitelli l'inmemory web api non dovrebbe ritornare i dati buildati perche' dovrebbe rispecchiare quello che succederebbe se avessimo un back-end. Il servizio specifico dovrebbe tornare i dati buildati ma non l'InMemory Web Api.

Reply:
Si effettivamente noi utilizziamo il metodo Build per costruirci dati a RunTime che ci servono solo nel front-end, quindi il back-end e tantomeno il database non devono avere queste informazioni